src/main/java/de/tum/in/www1/artemis/service/connectors/ContinuousIntegrationService.java

```diff
@@ -28,12 +28,13 @@
     /**
      * Creates the base build plan for the given programming exercise
      *
-     * @param exercise          a programming exercise with the required information to create the base build plan
-     * @param planKey           the key of the plan
-     * @param repositoryURL     the URL of the assignment repository (used to separate between exercise and solution)
-     * @param testRepositoryURL the URL of the test repository
+     * @param exercise              a programming exercise with the required information to create the base build plan
+     * @param planKey               the key of the plan
+     * @param repositoryURL         the URL of the assignment repository (used to separate between exercise and solution)
+     * @param testRepositoryURL     the URL of the test repository
+     * @param solutionRepositoryURL the URL of the solution repository
```

Comment:
please add a hint that this is only used for special cases

Reply:
I've added documentation.